web_widget_uom/static/src/js/widget.js

```diff
@@ -0,0 +1,64 @@
+/* Global jscolor */
+odoo.define('web.web_widget_uom', function(require) {
+    "use strict";
+
+    var basic_fields = require('web.basic_fields');
+    var field_registry = require('web.field_registry');
+    var session = require('web.session');
+    var rpc = require('web.rpc');
+
+    var FieldUoM = basic_fields.FieldFloat.extend({
+        /**
+         * Float fields have an additional precision parameter that is read from
+         * either the field node in the view or the field python definition itself.
+         *
+         * @override
+         */
+        init: function () {
```

Comment:
This init does not seem to be doing anything, the docstrings are pretty much copies of their super definitions so I believe you could remove these as well.